app/frontend/src/components/AvatarInput.tsx

```diff
@@ -131,7 +140,9 @@ export function AvatarInput({
           accept="image/*"
           id={id}
           type="file"
+          onClick={handleClick}
           onChange={handleChange}
+          ref={inputRef}
```

Comment:
Actually looking at [react hook form docs here on `useController`](https://react-hook-form.com/api#useController), shouldn't we be using the `ref` from the `field` object instead so value updates are managed by the lib rather than us having to manually clear the input value on cancel?

Reply:
In fact... do we even need a Controller? This is a native HTML input field so it's not a controlled component unlike Material UI fields. Wouldn't it be simpler to pass `register` in from the parent component with `useForm` and do `ref={register}` instead?